build.clj

```diff
@@ -0,0 +1,38 @@
+(ns build
+  (:require [clojure.tools.build.api :as b]))
+
+(def build-folder "target")
+(def jar-content (str build-folder "/classes"))
+
+(def native-basis (b/create-basis {:project "deps.edn"
+                                   :aliases [:native]}))
+(def basis (b/create-basis {:project "deps.edn"
+                            :aliases []}))
+(def app-name "jet")
+(def uber-file-name (format "%s/%s-standalone.jar" build-folder app-name)) ; path for result uber file
+
+(defn clean [_]
+  (b/delete {:path "target"})
+  (println (format "Build folder \"%s\" removed" build-folder)))
+
+(defn uber [_]
+  #_(clean nil)
+
+  (b/copy-dir {:src-dirs   ["resources"]         ; copy resources
+               :target-dir jar-content})
+
+  (b/javac {:src-dirs ["src-java"]
+            :class-dir jar-content
+            :basis native-basis
+            #_#_:javac-opts ["-Djet.native=true"]})
```

Comment:
Because you need to compile java code, the `build.clj` file is needed. But I want to add support for compiling java code in `clj-nix` (related to https://github.com/jlesquembre/clj-nix/issues/62) . Once it's implemented, you don't need a `build.clj` file for a relative standard Clojure build.

Reply:
Ya, that sounds great.  As I mentioned above, the jet project has a tiny bit of .java in it so this could also be a good test project for that.